charts/datadog/CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Datadog changelog
 
+# 2.25.0 
+
+* Introduce `process_agent.process_discovery` to configure `DD_PROCESS_AGENT_DISCOVERY_ENABLED`
```

Comment:
Should it be `processAgent.processDiscovery` ?

Reply:
Ah yes it should. Thanks for pointing that out!